extensions/quartz/deployment/src/test/java/io/quarkus/quartz/test/MisfirePolicyTest.java

```diff
@@ -0,0 +1,161 @@
+package io.quarkus.quartz.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.quartz.CronTrigger;
+import org.quartz.SchedulerException;
+import org.quartz.SimpleTrigger;
+import org.quartz.Trigger;
+import org.quartz.TriggerKey;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class MisfirePolicyTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset(
+                            "quarkus.quartz.\"simple_ignore_misfire_policy\".misfire-policy=IGNORE_MISFIRE_POLICY\n" +
```

Comment:
```suggestion
                            "quarkus.quartz.\"simple_ignore_misfire_policy\".misfire-policy=ignore-misfire-policy\n" +
```
The enums are hyphenated, so you could actually do as suggested. 
This can be applied throughout.

Looking at the test, I propose we change the order of the configuration key, to properly set the context. So instead of reading like `quarkus.quartz."<identity>.misfire-policy"`, it will read like `quarkus.quartz.misfire-policy."<identity>"`. I find the later to be easier to understand that we are dealing with the misfire configuration, especially when you have long job identity. 

The configuration could be something like
```java
/**
     * Misfire policy per job configuration.
     */
    @ConfigDocSection
    @ConfigDocMapKey("identity")
    @ConfigItem(name = "misfire-policy")
    public Map<String, QuartzNamedRuntimeConfig> misfirePolicyPerJob;

    @ConfigGroup
    public static class QuartzNamedRuntimeConfig {
        /**
         * The quartz misfire policy for this job.
         */
        @ConfigItem(defaultValue = "SMART_POLICY", name = ConfigItem.PARENT)
        public QuartzMisfirePolicy misfirePolicy;
    }
```

Reply:
Sounds good, but what if we have to add more per job parameters in the future ?

Reply:
The parameters will be prefixed by what we are looking to expose. Just like what I propose here.